### ISSUE TITLE
Singular fits in the wild

### ISSUE THREAD

**Issue by rebeccabieber** [LSC-MEM-course/random-effect-simulation-assignment-1-rebeccabieber#2]
I'm getting a singular fit when playing with random slopes for my data (which I'm sure I'm not handling properly anyway, since I'm using percent correct as my DV).  

Is it worth looking into that at this point, or not until I'm eventually running it properly (a glmer with binomial distribution and weighting - I think - though right now all that gets me is convergence issues)?

**Comment by scottrjackson** [comment]
singular fit is not necessarily a convergence issue. I'll pull and see what you're talking about

**Comment by scottrjackson** [comment]
oh, your data is not in the repo 😄 
Is this that data you can't share that you mentioned?  If so, maybe paste in some of the output, and tell me where you're getting singularity warnings. But we'll start discussing some convergence issues today, and I'll mention the difference between convergence and singularity.

**Comment by rebeccabieber** [comment]
Nope, I can share it! Just moved it into that repo.  I'm getting both
convergence and singular fit issues, depending on which random terms I
include

**Comment by rebeccabieber** [comment]
Nope, I can share it! Just moved it into that repo.  I'm getting both convergence and singular fit issues, depending on which random terms I include
